charts/stable/tailscale/Chart.yaml

```
@@ -33,4 +33,4 @@ sources:
   - https://github.com/truecharts/charts/tree/master/charts/stable/tailscale
   - https://hub.docker.com/r/tailscale/tailscale
 type: application
-version: 10.12.0
+version: 10.12.1
```

```suggestion
version: 10.12.0
```
You only modify docs. So not needed